### ISSUE TITLE
[PATCH] Remove a empty section warning

### ISSUE THREAD

**Issue by p5pRT** [Perl/perl5#11117]
Migrated from [rt.perl.org#83576](https://rt-archive.perl.org/perl5/Ticket/Display.html?id=83576) (status was 'rejected')

Searchable as RT83576$

**Comment by p5pRT** [comment]
### From @mstevens



---
 pod/perlunifaq\.pod |    2 \+\+
 1 files changed\, 2 insertions\(\+\)\, 0 deletions\(\-\)

<details><summary>Inline Patch</summary>

```diff
diff --git a/pod/perlunifaq.pod b/pod/perlunifaq.pod
index 9fd2b38..6bed02a 100644
--- a/pod/perlunifaq.pod
+++ b/pod/perlunifaq.pod
@@ -136,6 +136,8 @@ concern, and you can just C<eval> dumped data as always.
 
 =head2 Why do regex character classes sometimes match only in the ASCII range?
 
+See below.
+
 =head2 Why do some characters not uppercase or lowercase correctly?
 
 Starting in Perl 5.14 (and partially in Perl 5.12), just put a
-- 
1.7.3.1
```
</details>

**Comment by p5pRT** [comment]
### From @mstevens

Attaching patch\.

This is actually a bit icky so I'd respect anyone not wanthing to apply 
it\, but it makes podchecker happier\.

**Comment by p5pRT** [comment]
### From @mstevens

<details><summary>0001-Remove-a-empty-section-warning.patch</summary>

```diff
From 34d72917b5bc178bc45ffc67f222bb1e94c91334 Mon Sep 17 00:00:00 2001
From: Michael Stevens <mstevens@etla.org>
Date: Mon, 7 Feb 2011 17:12:33 +0000
Subject: [PATCH] Remove a empty section warning

---
 pod/perlunifaq.pod |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/pod/perlunifaq.pod b/pod/perlunifaq.pod
index 9fd2b38..6bed02a 100644
--- a/pod/perlunifaq.pod
+++ b/pod/perlunifaq.pod
@@ -136,6 +136,8 @@ concern, and you can just C<eval> dumped data as always.
 
 =head2 Why do regex character classes sometimes match only in the ASCII range?
 
+See below.
+
 =head2 Why do some characters not uppercase or lowercase correctly?
 
 Starting in Perl 5.14 (and partially in Perl 5.12), just put a
-- 
1.7.3.1


```
</details>

**Comment by p5pRT** [comment]
@mstevens - Status changed from 'new' to 'open'

**Comment by p5pRT** [comment]
### From @cpansprout

On Mon Feb 07 09&#8203;:16&#8203;:08 2011\, mstevens wrote&#8203;:
> Attaching patch\.
> 
> This is actually a bit icky so I'd respect anyone not wanthing to apply 
> it\, but it makes podchecker happier\.

I’m rather suprised that podchecker complains\. There’s nothing in the
POD spec forbidding adjacent headers\, as far as I can tell\. I’m
wondering whether podchecker should have some option for allowing that\.

So I’m going to mark this and \#83578 as rejected\. If anyone disagrees\,
I’ll reopen them\.

**Comment by p5pRT** [comment]
@cpansprout - Status changed from 'open' to 'rejected'

**Comment by p5pRT** [comment]
### From @mstevens

On Sun\, Feb 13\, 2011 at 01&#8203;:39&#8203;:42PM \-0800\, Father Chrysostomos via RT wrote&#8203;:
> On Mon Feb 07 09&#8203;:16&#8203;:08 2011\, mstevens wrote&#8203;:
> > Attaching patch\.
> > 
> > This is actually a bit icky so I'd respect anyone not wanthing to apply 
> > it\, but it makes podchecker happier\.
> 
> I’m rather suprised that podchecker complains\. There’s nothing in the
> POD spec forbidding adjacent headers\, as far as I can tell\. I’m
> wondering whether podchecker should have some option for allowing that\.
> 
> So I’m going to mark this and \#83578 as rejected\. If anyone disagrees\,
> I’ll reopen them\.

I can't see anything either in the spec\, I was going by what podchecker
says\.

I'd see about coming up with a patch to remove the warning in
Pod&#8203;::Parser\, but my understanding is that Pod&#8203;::Parser patches are
unlikely to be accepted\.

Michael

**Comment by p5pRT** [comment]
### From @cpansprout

On Sun Feb 13 13&#8203;:49&#8203;:48 2011\, mstevens wrote&#8203;:
> On Sun\, Feb 13\, 2011 at 01&#8203;:39&#8203;:42PM \-0800\, Father Chrysostomos via RT
wrote&#8203;:
> > On Mon Feb 07 09&#8203;:16&#8203;:08 2011\, mstevens wrote&#8203;:
> > > Attaching patch\.
> > > 
> > > This is actually a bit icky so I'd respect anyone not wanthing to
apply 
> > > it\, but it makes podchecker happier\.
> > 
> > I’m rather suprised that podchecker complains\. There’s nothing in the
> > POD spec forbidding adjacent headers\, as far as I can tell\. I’m
> > wondering whether podchecker should have some option for allowing that\.
> > 
> > So I’m going to mark this and \#83578 as rejected\. If anyone disagrees\,
> > I’ll reopen them\.
> 
> I can't see anything either in the spec\, I was going by what podchecker
> says\.
> 
> I'd see about coming up with a patch to remove the warning in
> Pod&#8203;::Parser\, but my understanding is that Pod&#8203;::Parser patches are
> unlikely to be accepted\.

Not in the perl core\, but if you submit them upstream
\(\<http&#8203;://rt\.cpan\.org/NoAuth/Bugs\.html?Dist=Pod\-Parser>\) they will be
included the next time Pod&#8203;::Parser is integrated\.
